.github/workflows/helm-version-update.yaml

```diff
@@ -0,0 +1,39 @@
+name: Build
+on:
+  schedule:
+    - cron: '0 8 * * 1'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+env:
+  HELM_VERSION: v3.11.1
+
+jobs:
+  ensure-versions:
+    runs-on: ubuntu-22.04
+    permissions:
+      pull-request: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3.5.0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v3.5
+        with:
+          version: ${{ env.HELM_VERSION }}
+
+      - name: Update Versions
+        run: |
+          ./.github/workflow/scripts/update-versions.sh
+          git diff
+
+      - name: Create Pull Request
+        id: cpr
+        uses: peter-evans/create-pull-request@v4
```

Comment:
Can we lock this action at the patch version, dependabot will manage updates for us

Reply:
Would be happy to. but not sure how to do that.

Reply:
```suggestion
        uses: peter-evans/create-pull-request@v5.0.0
```

Reply:
Ah. gotcha.